modin/experimental/xgboost/xgboost_ray.py

```diff
@@ -569,6 +547,21 @@ def _assign_row_partitions_to_actors(
     return row_parts_by_ranks
 
 
+def wait_computations(parts_ips_refs, parts_refs=[]):
```

Comment:
```suggestion
def wait_computations(obj_refs):
    ray.wait(obj_refs, num_returns=len(obj_refs))
```

Let's simplify this function.

Reply:
not actual